pkg/apis/kubeone/helpers.go

```diff
@@ -241,21 +241,23 @@ func (crc ContainerRuntimeConfig) CRISocket() string {
 	return ""
 }
 
-func (v VersionConfig) SandboxImage() (string, error) {
+func (v VersionConfig) SandboxImage(imageRegistry func(string) string) (string, error) {
```

Comment:
If we anyway always use `cluster.RegistryConfiguration.ImageRegistry` as an argument, maybe we don't need to take it as an argument and just use directly :shrug:

Reply:
Can you specify what you mean? `v VersionConfig` doesn't have access to `cluster.RegistryConfiguration.ImageRegistry` as far as I know.

Reply:
oh well :shrug: 
OK then